src/main/java/com/cascadebot/cascadebot/commands/moderation/BanCommand.java

```diff
@@ -0,0 +1,55 @@
+package com.cascadebot.cascadebot.commands.moderation;
+
+import com.cascadebot.cascadebot.commandmeta.CommandContext;
+import com.cascadebot.cascadebot.commandmeta.ICommandMain;
+import com.cascadebot.cascadebot.commandmeta.Module;
+import com.cascadebot.cascadebot.permissions.CascadePermission;
+import com.cascadebot.cascadebot.utils.DiscordUtils;
+import net.dv8tion.jda.core.Permission;
+import net.dv8tion.jda.core.entities.Member;
+import net.dv8tion.jda.core.exceptions.HierarchyException;
+import net.dv8tion.jda.core.exceptions.InsufficientPermissionException;
+import org.apache.commons.lang.ObjectUtils;
+
+public class BanCommand implements ICommandMain {
+
+    @Override
+    public void onCommand(Member sender, CommandContext context) {
+        String banLength;
+
+        if (context.getArgs().length == 0) {
+            context.replyDanger("Not enough arguments, please supply a username!");
+            return;
+        }
+
+        Member targetMember = DiscordUtils.getMember(context.getMessage(0), context.getGuild());
+        try {
+            context.getGuild().getController().ban(targetMember.getUser(), 7).queue();
+            context.replyInfo("**%s** has been banned!", targetMember.getUser().getAsTag());
+        } catch (NullPointerException e) {
```

Comment:
Instead of catching the NullPointerException, you should do a null check instead